libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp

```diff
@@ -73,15 +73,16 @@ void AP_Volz_Protocol::update()
             }
             
             // check if current channel PWM is within range
-            if (c->get_output_pwm() < c->get_output_min()) {
+            if (c->get_output_pwm() < PWM_POSITION_MIN) {
                 value = 0;
             } else {
-                value = c->get_output_pwm() - c->get_output_min();
+                value = c->get_output_pwm() - PWM_POSITION_MIN;
             }
 
             // scale the PWM value to Volz value
-            value = value + VOLZ_EXTENDED_POSITION_MIN;
-            value = value * VOLZ_SCALE_VALUE / (c->get_output_max() - c->get_output_min());
+           
```

Comment:
a small white space issue here.  We should remove the blank line under the comment and indent the lines to the same amount.